app/services/migration/users/merger.rb

```diff
@@ -0,0 +1,20 @@
+module Migration
+  module Users
+    class Merger
+      attr_reader :from_user, :to_user
+
+      def initialize(from_user:, to_user:)
+        @from_user = from_user
+        @to_user = to_user
+      end
+
+      def merge!
+        ApplicationRecord.transaction do
+          from_user.applications.update!(user: to_user)
+          from_user.participant_id_changes.update!(user: to_user) # not sure?
```

Comment:
I think this migrator happens after users so will be ok and not needed here

Reply:
@cwrw 
I wanted this service to be "complete", maybe we'll use it in other places.